topsort-endpoints.yml

```diff
@@ -210,25 +210,25 @@ components:
           example: v_89kjm
 
     Session:
-      description: The Session object allows correlating user activity during a session whether or not they are actually logged in. For all calls to Topsort, include as many as are known.
+      description: The Session object allows correlating user activity during a session whether or not they are actually logged in.
       required:
         - sessionId
       type: object
       properties:
         sessionId:
+          description: >
+            Long-lived token identifying the customer interacting with the marketplace.
+
+            If your users are always logged in you may use a hash of your customer ID, in which case the optional `consumerId` would not be needed.
+            If your users may interact with your app or site while logged out we recommend generating a random identifier (UUIDv4) on first load and store it on local storage (cookie, local storage, etc) and let it live for at least a year.
           type: string
-          example: 345bx7
+          example: ebeaf802-6d0a-41a3-ae59-661887c4f6cb
         consumerId:
```

Comment:
I was talking today with Felipe and he mentioned that we may not be taking into account the `consumerId` at all. This is important as we may want to avoid making false claims about what the API is doing.

Not sure though what the best course of action is:
1) remove the field, but that would prevent marketplaces from benefiting of cross device attribution when we implement it.
2) keep the field and change the language

If there are plans to implement this logic, then I would go with option 2.

Reply:
Seems like we don't so I'll do 1, new marketplaces will have to add it later.